src/main/scala/com.gu.contentapi.client/utils/QueryStringParams.scala

```diff
@@ -0,0 +1,21 @@
+package com.gu.contentapi.client.utils
+
+import java.net.URLEncoder
+
+import org.joda.time.ReadableInstant
+import org.joda.time.format.ISODateTimeFormat
+
+object QueryStringParams {
+  def apply(parameters: Iterable[(String, String)]) = {
```

Comment:
shouldn't this be `[String, Parameter]`? I don't think they are always strings, although they may have already been to-stringed at this point in which case fine :-)